sources/models/src/aws-k8s-1.30-nvidia/mod.rs

```diff
@@ -1,6 +1,6 @@
 use crate::{
     AutoScalingSettings, BootSettings, BootstrapContainer, CloudFormationSettings,
-    ContainerRuntimeSettings, DnsSettings, HostContainer, KubernetesSettings, MetricsSettings,
+    ContainerRuntimeSettings, DnsSettings, HostContainer, KubernetesSettings,
```

Comment:
You'll want to run `rustfmt` on all of these files, that will automatically sort the imports. Here I think that `NetworkSettings` should be on line 3